test/fuzz/fuzzer_pass_construct_composites_test.cpp

```diff
@@ -13,6 +13,7 @@
 // limitations under the License.
 
 #include "source/fuzz/fuzzer_pass_construct_composites.h"
+#include "source/fuzz/fuzzer_pass_replace_opselects_with_conditional_branches.h"
```

Comment:
Do we need this?

Reply:
No, I forgot to delete it after I quickly tried running the new fuzzer pass